### PR TITLE
I18n update for german

### PR DESCRIPTION
**Pull request by christophscheuing**
Improvement of the German language version
(in the code + addition of the file "Beispielconfig auf Deutsch.json")

### CODE REVIEW

File: src/app/core/config/Beispielconfig auf Deutsch.json [hunk 1]
```diff
@@ -0,0 +1,1634 @@
...
+    "view:report": {
+        "component": "Reporting",
+        "config": {
+            "reports": [
+                {
+                    "title": "Basisbericht",
+                    "aggregationDefinitions": [
+                        {
+                            "query": "Child:toArray[*isActive=true]",
+                            "label": "All children",
```
Comment (TheSlimvReal) [inline src/app/core/config/Beispielconfig auf Deutsch.json:1353]:
```suggestion
                            "label": "Kinder"
```

File: src/app/core/config/Beispielconfig auf Deutsch.json [hunk 2]
```diff
@@ -0,0 +1,1634 @@
...
+                                },
+                                {
+                                    "label": "Mädchen",
+                                    "query": "[*gender=W]"
+                                }
+                            ]
+                        },
+                        {
+                            "query": "School:toArray",
+                            "label": "Alle Schulen",
```
Comment (TheSlimvReal) [inline src/app/core/config/Beispielconfig auf Deutsch.json:1367]:
```suggestion
                            "label": "Schulen",
```

File: src/app/core/config/Beispielconfig auf Deutsch.json [hunk 3]
```diff
@@ -0,0 +1,1634 @@
...
+                            "aggregations": [
+                                {
+                                    "label": " Auf einer Schule",
```
Comment (TheSlimvReal) [inline src/app/core/config/Beispielconfig auf Deutsch.json:1370]:
```suggestion
                                    "label": "Schulgänger:innen"
```

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Looks very good, just some small typos

File: src/app/core/config/Beispielconfig auf Deutsch.json (hunk 1 above)
Comment (christophscheuing) [reply src/app/core/config/Beispielconfig auf Deutsch.json:1353]: I think the guideline was to rather speak of "Schüler:innen" instead of "children".

**Review by TheSlimvReal — APPROVED** [review]

**Pull request merged**